gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineFormatIntegrationTest.groovy

```diff
@@ -59,16 +69,13 @@ class BaselineFormatIntegrationTest extends AbstractPluginTest {
     def 'format task fixes styles'() {
         when:
         buildFile << standardBuildFile
-        file('src/main/java/test/Test.java') << validJavaFile
+        file('src/main/java/test/Test.java') << invalidJavaFile
 
         then:
         BuildResult result = with('format').build();
         result.task(":format").outcome == TaskOutcome.SUCCESS
         result.task(":spotlessApply").outcome == TaskOutcome.SUCCESS
-        file('src/main/java/test/Test.java').text == '''
-            package test;
-            public class Test { void test() {} }
-        '''.stripIndent()
+        file('src/main/java/test/Test.java').text == validJavaFile
```

Comment:
think this test used to verify that the unused `import com.java.tests` was removed, so probably want to keep that around

Reply:
Added it back as `import com.java.unused;` to be explicit